index.html

```diff
@@ -1036,10 +1036,8 @@ <h3>Concrete Lifecycle Example</h3>
 Implementers that are interested in understanding more about the
 <code>proof</code> mechanism used above can learn more in Section <a
 href="#proofs-signatures"></a> and by reading the following specifications:
-Data Integrity [[VC-DATA-INTEGRITY]], Linked Data Cryptographic Suites
-Registry [[?LDP-REGISTRY]], and JSON Web Signature (JWS) Unencoded Payload Option
-[[RFC7797]]. A list of proof mechanisms is available in the Verifiable
-Credentials Extension Registry [[VC-EXTENSION-REGISTRY]].
+Data Integrity [[VC-DATA-INTEGRITY]] and the "Proofs" section of the Verifiable
```

Comment:
```suggestion
Securing Verifiable
Credentials using JSON Web Tokens [VC-JWT], Data Integrity [[VC-DATA-INTEGRITY]] and the "Proofs" section of the Verifiable
```

Reply:
I agree with Orie's suggestion.

Reply:
I didn't think that VC-JWT used `<code>proof</code>`. If it does, then I agree it makes sense to reference it here. Otherwise, why would it be mentioned in a section about "Implementers that are interested in understanding more about the `<code>proof</code>` mechanism" ?

Reply:
@OR13, @selfissued, and @mprorock ... as @dlongley noted, VC-JWT doesn't use `proof`, that's why the reference was removed. VC-JWT is referred to, normatively, in other sections of the specification such as:

* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#dfn-conforming-processor
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proofs-signatures
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#example-usage-of-the-proof-property-on-a-verifiable-credential
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#media-types
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#proof-formats
* https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1086.html#base-context
* ... and so on

The proposed change makes no sense, VC-JWT doesn't use `proof`. This section is about `proof`.

Reply:
vc-jwt DOES use `proof`... https://w3c.github.io/vc-data-model/#proofs-signatures

Its just not super clear how, because of this:
<img width="925" alt="Screen Shot 2023-04-27 at 12 41 23 PM" src="https://user-images.githubusercontent.com/8295856/234946405-770a4a31-9d63-46ac-952a-294573dc3ac8.png">

Reply:
We could probably file an issue for this, and merge the PR as is.

Reply:
As a follow up, since `proof` is allowed in `vc+ld+json`, it is assumed to be allowed in JWTs... so we can't hand wave over this, and the spec text we have today is now totally out of date with the latest resolutions regarding media types and proof.

Reply:
Ok, I'll do that. 

Perhaps that section should be titled: "Securing Verifiable Credentials", and then we talk about external proofs and embedded proofs, and normatively point to both securing format specifications.

Reply:
Issue raised to track the concern here: https://github.com/w3c/vc-data-model/issues/1104